design/lpar-status.md

```diff
@@ -31,70 +33,113 @@ This section describes the actual behavior of the "Activate LPAR" and
+* Status = `status` property of the LPAR object before the operation
+* Activation = what happens related to activation when the activation operation
+  is performed
+* Load = what happens related to load when the activation operation is performed
+* Load parameters = where are the parameters for the load taken from
+
+Values:
+
+* omitted = this optional operation parameter has not been provided
+* other = any other value than those in the table rows above
+* any = the property or parameter can have any valid value and is not relevant
+  for this case
+
+Some more notable details about some of these cases:
+
+* If the LPAR has status 'not-operating', the activation behavior depends on
+  whether a load profile or an image profile is used:
 
   - If an image profile is used, the LPAR will be deactivated and reactivated
-    and dependent on the 'load-at-activation' property, it will also be loaded.
+    (and possibly loaded).
     The deactivation happens always and does not dependent on whether the
     activation parameters in the image profile have changed.
 
   - If a load profile is used, the LPAR remains active and it will just be
     loaded.
 
-    **TBD: Does the loading in this case only happen when 'load-at-activation=True'?**
-
-* If the LPAR has status 'operating' and 'force=False' is specified or
-  defaulted, the "Activate LPAR" operation will fail, regardless of whether an
-  image or load profile or no profile has been specified.
-
 * If the LPAR has status 'operating' and 'force=True' is specified, the behavior
   depends on whether a load profile or an image profile is used:
 
   - If an image profile is used, the control program in the LPAR will get the
     shutdown signal so it can perform an orderly shutdown. After the control
-    program has stopped, the LPAR will be deactivated and reactivated and
-    dependent on the 'load-at-activation' property, it will be loaded. The
-    deactivation happens always and does not dependent on whether the activation
-    parameters in the image profile have changed.
+    program has stopped, the LPAR will be deactivated and reactivated (and
+    possibly loaded). The deactivation happens always and does not dependent
+    on whether the activation parameters in the image profile have changed.
 
   - If a load profile is used, the control program in the LPAR will get the
     shutdown signal so it can perform an orderly shutdown. After the control
     program has stopped, the LPAR remains active and will be loaded again.
 
-Notes:
-
 * If a load is performed, the resulting LPAR status will go to 'operating' if
   the loading has succeeded and the control program got started. If the load
   parameters do not specify a valid control program, or if the load parameters
   are not valid for the control program, the LPAR status will go to
   'not-operating'.
 
-### Behavior of "Load LPAR" operation
+### Behavior of "Load Logical Partition" operation
```

Comment:
This header is misleading, because you're still talking about what is "under the hood" the 'Activate" operation. Load Logical Partition in the API is a separate operation that refers to an ECKD-CCW load

Reply:
Actually this section does want to talk about the "Load Logical Partition" operation. The implicit load done as part if the "Activate Logical Partition" operation is covered n the same-named chapter.

Reply:
OK. Then it seems a bit confusing to name just one of the load operations then? Unless you're referring to the naming of one of your modules, in which case, sorry for my confusion.

Reply:
The load operations are meant to be the HMC operations. At this point, the document only describes the "Load Logical Partition" HMC operation, because that is the one the zhmc_lpar Ansible module uses to perform a load, so it is of greater interest than the others. I am aware of the newer "Load" HMC operation and at some point the zhmc_lpar Ansible module will migrate to use that one.

Reply:
If you're only using Load Logical Partition, does that mean that you're not presently handling SCSI or NVMe loads right now? That would be SCSI Load / SCSI Dump / NVMe Load / NVMe Dump.

Thrilled to hear you'll be migrating to the new Load (`POST /api/logical-partitions/{logical-partition-id}/operations/load-program`)  operation

Reply:
There are zhmcclient methods for SCSI and NVMe load and dump. Just this particular document does not cover them.

The only load operation not yet supported is the newer "load-program".